hr_employee_service_contract/models/hr_employee.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    first_contract_id = fields.Many2one(
+        'hr.contract',
+        readonly=True,
+        compute='_compute_first_contract_id',
+        string='First Contract',
+        help='First contract of the employee',
+    )
+    last_contract_id = fields.Many2one(
+        'hr.contract',
+        readonly=True,
+        compute='_compute_last_contract_id',
+        string='Last Contract',
+        help='Last contract of the employee',
+    )
+    service_start_date = fields.Date(
+        string='Start Date',
+        readonly=True,
+        related='first_contract_id.date_start',
+    )
+    service_termination_date = fields.Date(
+        string='Termination Date',
+        readonly=True,
+        related='last_contract_id.date_end',
+    )
+
+    @api.multi
+    @api.depends('contract_ids')
+    def _compute_first_contract_id(self):
+        Contract = self.env['hr.contract']
+        for employee in self:
+            employee.first_contract_id = Contract.search([
+                ('employee_id', '=', employee.id),
+                ('state', 'not in', self._service_contract_states()),
+            ], order='date_start asc', limit=1)
+
+    @api.multi
+    @api.depends('contract_ids')
+    def _compute_last_contract_id(self):
+        Contract = self.env['hr.contract']
+        for employee in self:
+                employee.last_contract_id = Contract.search([
+                    ('employee_id', '=', employee.id),
+                    ('state', 'not in', self._service_contract_states()),
+                ], order='date_end desc', limit=1)
+
+    @api.multi
+    @api.onchange('service_hire_date')
+    def _onchange_service_hire_date(self):  # pragma: no cover
```

Comment:
You can remove this method because  method is not in use

Reply:
It's in use actually, in parent module: https://github.com/OCA/hr/blob/f0f17b3149ceaf2cf722d757385357556ffefac6/hr_employee_service/models/hr_employee.py#L107